### DISCUSSION TITLE
My projects list is always empty

### DISCUSSION THREAD

**Discussion by jeusdi** [LazyVim/LazyVim#2059] [Q&A]
Should I do something to feed project Telescope list?

It's always empty.

I've loaded maven projects and lsp is attached and working right. But project list is always empty.
Any ideas?

**Comment by dpetka2001** [comment]
See #1978, but keep in mind that this was chosen to be the default behavior in LazyVim, because it changes the `cwd` and many users were having problem with it. So, if a user enables it, it's on his own accord.
